tests/cases/Integration/SendSms.phpt

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types = 1);
```

Comment:
Slo by to hodit do tests/Cases/E2E? U jinych repozitaru to tak delam. Kdyz je to napojene na realnou sluzbu.

Reply:
Jakože takto? tests/cases/E2E/SendSms.phpt

Reply:
Adresář cases, přejmenuju na Cases, ok?

Reply:
Ok ok